samples/CommunityToolkit.Maui.Sample/ViewModels/Converters/ByteArrayToImageSourceConverterViewModel.cs

```diff
@@ -50,15 +50,17 @@ async Task DownloadDotNetBotImage(CancellationToken token)
 
 		try
 		{
+#pragma warning disable S1075 // URIs should not be hardcoded
 			DotNetBotImageByteArray = await client.GetByteArrayAsync("https://user-images.githubusercontent.com/13558917/137551073-ac8958bf-83e3-4ae3-8623-4db6dce49d02.png", maximumDownloadTimeCTS.Token).WaitAsync(token).ConfigureAwait(false);
+#pragma warning restore S1075 // URIs should not be hardcoded
 
 			await minimumDownloadTimeTask.ConfigureAwait(false);
 
 			LabelText = "The above image was downloaded as a byte[]";
 		}
 		catch (Exception e)
 		{
-			Trace.WriteLine(e);
+			Trace.TraceError(e.StackTrace);
```

Comment:
I think we should rely on the full information

```suggestion
			Trace.TraceError(e);
```

Reply:
That would cause an error and the application would fail to compile with that change. `CS1503` cannot convert from `System.Exception` to `string?` I will work on making an appropriate change and update the PR.

Reply:
Then you can use `ex.ToString()`

Reply:
How about `Trace.TraceError("{0}", e);`

Reply:
Yes that also works

Reply:
Thank you for catching these errors and helping me fix them. I appreciate the time you take to review this.